Cargo.toml

```diff
@@ -20,8 +20,7 @@ codegen-units = 1
 
 [patch.crates-io]
 # In development.
-halo2 = { git = "https://github.com/zcash/halo2.git", rev = "a7cd600eb60b1528159b92af5e426adcc615de1a" }
-orchard = { git = "https://github.com/zcash/orchard.git", rev = "8779ce8f1a638ebbc9b229d4eff3a29ef4de7ac0" }
+orchard = { git = "https://github.com/zcash/orchard.git", rev = "2c8241f25b943aa05203eacf9905db117c69bd29" }
```

Comment:
I've checked that this commit merges the circuit bugfixes in https://github.com/zcash/orchard/pull/209.